src/content/blog/podman-and-docker-rootless.md

```diff
@@ -0,0 +1,481 @@
+
+## Configuring Podman for DDEV
+
+After installing Podman, enable the API socket so DDEV can connect.
+
+### Podman Rootless on Linux
+
+This is the recommended configuration for most users.
+
+1. Enable and start the Podman socket:
+
+   ```bash
+   systemctl --user enable --now podman.socket
+   ```
+
+2. Verify the socket is running:
+
+   ```bash
+   ls $XDG_RUNTIME_DIR/podman/podman.sock
+   ```
+
+   You should see `/run/user/1000/podman/podman.sock` (the number may vary).
+
+   You can also check the socket path with:
+
+   ```bash
+   podman info --format '{{.Host.RemoteSocket.Path}}'
+   ```
+
+3. Configure Docker to use Podman. Choose one of these methods:
+
+   **Option A: Create Docker context** (if you have `docker` CLI installed):
+
+   ```bash
+   # View existing contexts
+   docker context ls
+
+   # Create Podman rootless context
+   docker context create podman-rootless \
+       --description "Podman (rootless)" \
+       --docker host="unix://$XDG_RUNTIME_DIR/podman/podman.sock"
+
+   # Switch to the new context
+   docker context use podman-rootless
+
+   # Verify it works
+   docker ps
+   ```
+
+   **Option B: Set environment variable in your shell profile** (`~/.bashrc` or `~/.zshrc`):
+
+   ```bash
+   export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
+   ```
+
+For additional details, see the [Podman socket activation documentation](https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md).
+
+### Podman Rootful on Linux
+
+Rootful Podman requires configuring user group permissions, similar to [Docker's Linux post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/).
+
+1. Create the `podman` group:
+
+   ```bash
+   sudo groupadd podman
+   ```
+
+2. Add your user to the group:
+
+   ```bash
+   sudo usermod -aG podman $USER
+   ```
+
+3. Log out and log back in, or activate the changes immediately:
+
+   ```bash
+   newgrp podman
+   ```
+
+   If you're in a VM, restart it for changes to take effect.
+
+4. Configure Podman socket permissions:
+
+   ```bash
+   # Configure permissions for /run/podman
+   sudo tee /etc/tmpfiles.d/podman.conf > /dev/null <<EOF
+   d /run/podman 0770 root podman
+   EOF
+
+   # Configure socket to run with podman group
+   sudo mkdir -p /etc/systemd/system/podman.socket.d
+   sudo tee /etc/systemd/system/podman.socket.d/override.conf > /dev/null <<EOF
+   [Socket]
+   SocketMode=0660
+   SocketUser=root
+   SocketGroup=podman
+   EOF
+
+   # Apply changes
+   sudo systemd-tmpfiles --create
+   sudo systemctl daemon-reload
+   ```
+
+5. Enable and start the socket:
+
+   ```bash
+   sudo systemctl enable --now podman.socket
+   ```
+
+6. Verify the socket:
+
+   ```bash
+   ls /var/run/podman/podman.sock
+   ```
+
+   Or check with:
+
+   ```bash
+   sudo podman info --format '{{.Host.RemoteSocket.Path}}'
+   ```
+
+7. Configure Docker to use Podman. Choose one of these methods:
```

Comment:
This statement will be confusing. I assume this means "Configure `docker` CLI to connect to Podman instead of Docker Daemon."